File: JAVA PHASE GATE/Day 2/GroceryApp.java
import java.util.Arrays;
import java.util.Scanner;
    public class GroceryApp {
        public static void main(String...Odili){
            Scanner input = new Scanner(System.in);
            Scanner input2 = new Scanner(System.in);
            Scanner input3 = new Scanner(System.in);


    

String menu =("""
FLASH GROCERY

1. Add item

2. Remove item

3. Show all available items

""");

System.out.println(menu);
String menuOption = input.nextLine();

    switch(menuOption){
        case "1": 
            
            int length = 0;
            String condition = "no";
            String answer = "";

           System.out.println("Enter the length of the item you want to add");
                length = input.nextInt();

            String [] add = new String [length];
         
        
           for(int count = 0; count < length; count ++){

            System.out.println("Enter item");
                add[count]  = input2.nextLine();

             System.out.println(Arrays.toString(add));


            }
           
       }
       

    












           
            
            int length = 0;
            String condition = "no";
            String answer = "";

           System.out.println("Enter the length of the item you want to add");
                length = input.nextInt();

            String [] add = new String [length];
         
        do{
           for(int count = 0; count < length; count ++){

            System.out.println("Enter item");
                add[count]  = input2.nextLine();

            

            }
            
        }

        while(answer.equalsIgnoreCase(condition));
        
        System.out.println(Arrays.toString(add));

   
        String [] remove = new String [length];

     System.out.println("Enter the length of item to remove");
        length = input.nextInt();       

    do{
      
        for(int count = 0; count < length; count ++){
        
        System.out.println("Enter item to remove");
            remove[count] = input2.nextLine();
           if(remove.equals(add[0])){
                System.out.println(Arrays.toString(remove));
            }
        }
      }


        
        while(answer.equalsIgnoreCase(condition));

    
         System.out.println(Arrays.toString(add));

        System.out.println("Showing All Items");
        System.out.println(Arrays.toString(add));





   
         
      }
  }

File: PYTHON PHASE GATE/Day 2/grocery_app.py
from grocery_app_function import remove_items, add_items, show_all_items

def main_menu(my_list):

    menu_app = ("""

    GROCERY LIST
    
    1. Add item
    2. Remove item
    3. Show all available items
    """)
    
    print(menu_app)
    menu_option = input("Enter any option ")

    match(menu_option):
        case "1": 
                add = input("Enter item to add ")
                add_items(my_list, item)
                print("Item added Successfully")


        case "2": 
                remove = input("Enter item to remove ")
                remove_items(my_list, item)
                print("Item removed Successfully")

        

        case "3":
                print("Showing all available items ", my_list) 






def main():
    my_list = ["garri", "rice", "yam"]
    main_menu(my_list)


    
main()   









    
